File: ProgramingBasics/SimpleLoops/04.SumNumbers/04.SumNumbers/SumNumbers.cs
using System;

namespace _04.SumNumbers
{
    public class SumNumbers
    {
        public static void Main(string[] args)
        {
            var n = int.Parse(Console.ReadLine());
            var sum = 0;
            var num = 0;

            for (var i = 0; i < n; i++)
            {
                num = int.Parse(Console.ReadLine());
                sum += num;
            }

            Console.WriteLine(sum);
        }
    }
}

File: .ProgramingBasics/Simple-Loops/04.SumNumbers/src/SumNumbers.java
import java.util.Scanner;

public class SumNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        int sum = 0;

        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(scanner.nextLine());
            sum += num;
        }

        System.out.println(sum);
    }
}
